SU2_CFD/src/python_wrapper_structure.cpp

```diff
@@ -367,68 +306,50 @@ bool CDriver::ComputeVertexForces(unsigned short iMarker, unsigned long iVertex)
   iPoint = geometry_container[ZONE_0][INST_0][MESH_0]->vertex[iMarker][iVertex]->GetNode();
 
   /*--- It is necessary to distinguish the halo nodes from the others, since they introduce non physical forces. ---*/
-  if(geometry_container[ZONE_0][INST_0][MESH_0]->nodes->GetDomain(iPoint)) {
-    /*--- Get the normal at the vertex: this normal goes inside the fluid domain. ---*/
-    Normal = geometry_container[ZONE_0][INST_0][MESH_0]->vertex[iMarker][iVertex]->GetNormal();
-    AreaSquare = 0.0;
-    for(iDim = 0; iDim < nDim; iDim++) {
-      AreaSquare += Normal[iDim]*Normal[iDim];
-    }
-    Area = sqrt(AreaSquare);
+  if (!geometry_container[ZONE_0][INST_0][MESH_0]->nodes->GetDomain(iPoint))
+    return true;
 
-    /*--- Get the values of pressure and viscosity ---*/
-    Pn = solver_container[ZONE_0][INST_0][MESH_0][FLOW_SOL]->GetNodes()->GetPressure(iPoint);
-    if (viscous_flow) {
-      Viscosity = solver_container[ZONE_0][INST_0][MESH_0][FLOW_SOL]->GetNodes()->GetLaminarViscosity(iPoint);
-    }
+  /*--- Get the normal at the vertex: this normal goes inside the fluid domain. ---*/
+  Normal = geometry_container[ZONE_0][INST_0][MESH_0]->vertex[iMarker][iVertex]->GetNormal();
+  Area = GeometryToolbox::Norm(nDim, Normal);
 
-    /*--- Calculate the inviscid (pressure) part of tn in the fluid nodes (force units) ---*/
-    for (iDim = 0; iDim < nDim; iDim++) {
-     PyWrapNodalForce[iDim] = -(Pn-Pinf)*Normal[iDim];     //NB : norm(Normal) = Area
-    }
+  /*--- Get the values of pressure and viscosity ---*/
+  Pn = solver_container[ZONE_0][INST_0][MESH_0][FLOW_SOL]->GetNodes()->GetPressure(iPoint);
```

Comment:
This is a bit different from the function compute_vertex_traction in Csolver.cpp. For example, the Pinf there is equal to config->GetPressure_FreeStreamND and everything is later multiplied for a factor accounting for adimensionalisation. Should this part be modified to be the same?

Reply:
In any case, if we now compute all the tractions inside SU2 itself, it there still the need for this function?
I think it is only used in an old test case...

Reply:
If it is not too much work adapt the old case we can remove this.

Reply:
The old test case uses a FSIdriver that I cannot find anymore, not sure if it would work in any case

Reply:
My bad, the test case is indeed used, not with this FSIdriver, but with the singlezone. It is a forced response.
I will try update the test case to the new routines